src/Aeon.Acquisition/Aeon.Acquisition.csproj

```diff
@@ -30,7 +30,7 @@
     <PackageReference Include="Bonsai.Design" Version="2.7.0" />
     <PackageReference Include="Bonsai.System" Version="2.7.0" />
     <PackageReference Include="Bonsai.Scripting.Expressions" Version="2.7.0" />
-    <PackageReference Include="LibGit2Sharp" Version="0.26.2" />
```

Comment:
Why did you downgrade this lib?

Reply:
Some questionable decisions on the newer versions of the package were made.
[libgit2/libgit2sharp#2020](https://github.com/libgit2/libgit2sharp/pull/2020)

Reply:
@bruno-f-cruz already replied, but I should have made these changes into a separate commit, will fix it now